spark-cypher-testing/src/test/scala/org/opencypher/spark/api/io/sql/SqlPropertyGraphDataSourceTest.scala

```diff
@@ -518,4 +518,44 @@ class SqlPropertyGraphDataSourceTest extends CAPSTestSuite with HiveFixture with
     val e = the [SqlDataSourceConfigException] thrownBy pgds.graph(GraphName("g"))
     e.getMessage should (include("unknown") and include("known1") and include("known2"))
   }
+
+  it("reads nodes and rels from file-based sources") {
+    val ddlString =
+      s"""
+         |CREATE GRAPH TYPE personSchema (
+         | Person (id INTEGER, name STRING),
+         | KNOWS,
+         | (Person),
+         | (Person)-[KNOWS]->(Person)
+         |)
+         |CREATE GRAPH fooGraph OF personSchema (
```

Comment:
We could simplify by inlining the graph type here.

```suggestion
         |CREATE GRAPH fooGraph (
         | Person (id INTEGER, name STRING),
         | KNOWS,
         |
```
batch 1